### ISSUE TITLE
How to count the occurrences of a perticular element in the list?

### ISSUE THREAD

**Issue by AshishNamdev** [AshishNamdev/iqa#20]
How to count the occurrences of a perticular element in the list?

- In Python list, we can count the occurences of a individual element by using a `count()` function.

### Example # 1:
```python
weekdays = [‘sun’,’mon’,’tue’,’wed’,’thu’,’fri’,’sun’,’mon’,’mon’]
print(weekdays.count(‘mon’))
```
### Output: 
```bash
3
```
### Example # 2:
```python
weekdays = [‘sun’,’mon’,’tue’,’wed’,’thu’,’fri’,’sun’,’mon’,’mon’]
print([[x,weekdays.count(x)] for x in set(weekdays)])
```
### Output:
```bash
[[‘wed’, 1], [‘sun’, 2], [‘thu’, 1], [‘tue’, 1], [‘mon’, 3], [‘fri’, 1]]
```

**Comment by AshishNamdev** [comment]
Included in [Q9](../blob/master/questions/Q1-50.md#Q9)

**Comment by AshishNamdev** [comment]
included in commit 0ee53c12045d5ea4788e9b4527a052dd89c4cfdd
